packages/xo-web/src/xo-app/jobs/overview/index.js

```diff
@@ -144,19 +145,15 @@ export default class Overview extends Component {
     }
   }
 
-  _getScheduleJob(schedule) {
-    const { jobs } = this.state || {}
-    return jobs[schedule.jobId]
-  }
-
   _getIsScheduleUserMissing = createSelector(
     () => this.state.schedules,
     () => this.props.users,
-    (schedules, users) => {
+    () => this.state.jobs,
+    (schedules, users, jobs) => {
       const isScheduleUserMissing = {}
 
       forEach(schedules, schedule => {
-        isScheduleUserMissing[schedule.id] = !find(users, user => user.id === this._getScheduleJob(schedule).userId)
+        isScheduleUserMissing[schedule.id] = !find(users, user => user.id === jobs[schedule.jobId].userId)
```

Comment:
`jobs[schedule.jobId]` might be `undefined`, right?

Reply:
Yes but:
- the bug existed before
- it can no longer happen after 841498b3a81f0e52ffaccbbdb6c2a916f77fefb2 because only schedules with jobs will be processed